### PR TITLE
Hoard of metalware_salomon_richard

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "A Hoard of Inscribed Gandharan Metalware"
+authors:
+  - "Richard Salomon"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3291581&url=article"
```
Comment (khemarato) [inline _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md:5]: Maybe save our users one click by using this instead: https://poj.peeters-leuven.be/content.php?id=3291581&url=article&download=yes

File: _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/15iT7pBJA3fO7dGUrfuC7Ve3MPa_yGagk/view?usp=sharing"
+course: medieval
+tags:
+  - indian
+  - archeology
+year: 2022
+journal: jiabs
+publisher:
```
Comment (khemarato) [inline _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md:14]: remove empty fields 😊

File: _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: "A Hoard of Inscribed Gandharan Metalware"
+authors:
+  - "Richard Salomon"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3291581&url=article"
+drive_links:
+  - "https://drive.google.com/file/d/15iT7pBJA3fO7dGUrfuC7Ve3MPa_yGagk/view?usp=sharing"
+course: medieval
```
Comment (khemarato) [inline _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md:8]: 🧐  Maybe this is a bit early for "medieval"? Perhaps should be moved to "sects"?

File: _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: "A Hoard of Inscribed Gandharan Metalware"
+authors:
+  - "Richard Salomon"
```
Comment (khemarato) [inline _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md:4]: `salomon-richard` in the filename

File: _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md (hunk 3 above)
Comment (JJM317) [reply _content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md:8]: agreed

**Comment by JJM317** [comment]
Hi Khemarato,

I agree this is too early for Medieval and made the course change to sect,
however, I feel like sect isn't quite right either. Could Central Asian or
Gandharan be a stand alone topic, just like East Asian Buddhism? BTW,
Medieval Buddhism seems a bit arbitrary. How is it being defined? Are you
going off of Medieval India dating, approximately 6th century to 13th
century?

These thoughts came to me upon seeing that the next item in the Medieval
folder is an art book on Gandhara and the Silk Road, with several art
pieces coming before the 6th century.

Thanks,
~Joe

**Pull request merged**